packaging/docker/entrypoint.py

```diff
@@ -170,6 +171,29 @@ def load_config() -> List[str]:
     logger.error('Failed to start nginx')
     exit(1)
 
+def graceful_exit(signal, frame):
+    logger.info(f'Received signal {signal}. Exiting gracefully')
+    if rotki.poll() is None:
+        rotki.send_signal(SIGTERM)
```

Comment:
Why work with `send_signal` and not use `process.terminate`? For unix it's the same (according to docs), but for windows it uses windows specific code as those signals are unix specific.

Though since this is docker, it would only run in unix, right?

Also after `send_signal()` shouldn't you wait somewhere that the signal is handled? Or does `send_signal()` block?

Reply:
Yes, I checked the underlying code and is SIGTERM is provided it calls `.terminate`, so using `.terminate` probably makes more sense.

> Though since this is docker, it would only run in unix, right?

Yeah, this is also true, but I changed it to `.terminate` anyways.

> Also after send_signal() shouldn't you wait somewhere that the signal is handled? Or does send_signal() block?

Thanks for attracting attention to this! Originally I did a quick check and decided that send_signal is synchronous, but now checked once again and see that it is not synchronous. Will fix this.